aioshelly/rpc_device/device.py

```diff
@@ -533,3 +537,26 @@ def _parse_dynamic_components(self, components: dict[str, Any]) -> None:
                 for item in self._dynamic_components
             }
         )
+
+    async def _retrieve_blutrv_components(self, components: dict[str, Any]) -> None:
```

Comment:
```suggestion
    async def _retrieve_blu_components(self, components: dict[str, Any]) -> None:
```

I think it would be better not to tie this only with TRV so we are ready to support other BLU devices in the future

Reply:
Logic so far is quite tie to the `blutrv`. We can always update it once we have new devices.
After all is an internal method, not a public one.